test/cases/regression.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# Provision the software under tet.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+BLUEPRINT_FILE=/tmp/blueprint.toml
+COMPOSE_START=/tmp/compose-start.json
+COMPOSE_INFO=/tmp/compose-info.json
+
+# Write a basic blueprint for our image.
+tee "$BLUEPRINT_FILE" > /dev/null << EOF
+name = "redhat-lsb-core"
+description = "A base system with redhat-lsb-core"
+version = "0.0.1"
+
+[[packages]]
+name = "redhat-lsb-core"
+
+[[packages]]
+name = "nss"
```

Comment:
I would like a comment here that explains that `nss` is excluded from RHEL 8.4 and this test proves that it can re-added into a package set using a blueprint.

Reply:
Fixed :+1: